src/hooks/useStableSwapEstimateRemoveLiquidity.ts

```diff
@@ -58,6 +60,10 @@ export default function useStableSwapEstimateRemoveLiquidity({
   const getEstimatedAmounts = useCallback(async () => {
     setError(null)
 
+    if (amount?.greaterThan(lpTokenBalance ?? BIG_INT_ZERO)) {
+      setError({ ...new Error(), reason: 'Insufficient LP amount' })
+    }
```

Comment:
I don't think this is an error, per se, it should just return, this is an invalid input